docs/eth.md

```diff
@@ -73,12 +73,6 @@ Sent exit transaction
 Transaction Hash: 0x4df9c79d17036b7468b69f83a313a1fb87ccc987e9e28be1e78f81d2d19afab8
 ```
 
-Exiting a utxo without trust-node:
```

Comment:
WIll this be added in a future PR or is it removed permanently? Also if the command for exitting fees should be included (if it is the same as a deposit or regular utxo exit, this should be explicitly stated.)